app/views/users/show.html.erb

```diff
@@ -36,6 +36,18 @@
             <%= c.user_count%>
           </div>
         </div>
+        <div class='col'>
+          <div class='course-details_title'>
+            Role
+          </div>
+          <div class='course-details_value'>
+            <% r = c.courses_users %>
+            <% r.each do |p| %>
+              <% @pass = p.role %>
+            <%end%>
+            <%= role(@pass) %>
```

Comment:
Thanks it's sorted now ,loop is removed . I ll convert the file to haml in next pull request and also add controller tests.